flutter-app/flutter_drawing_app/starter/lib/drawing_page.dart

```diff
@@ -286,12 +286,16 @@ Widget buildUploadButton() {
       );
       } 
     }
+    //
     return Container(
+      width: 0.95 * width,
       decoration: BoxDecoration(
-        color: Colors.red[200]
+        image: DecorationImage(
+          fit: BoxFit.fill,
+          image: AssetImage("images/hand-xray.jpeg"),
```

Comment:
Can we make a constant for this value

Reply:
Ditto! Good comment. This can be done for another variables too. Lets push it to its own PR and merge this in byitself